.slim-lint.yml

```diff
@@ -5,5 +5,4 @@ linters:
   LineLength:
     max: 140
     exclude:
-      - 'app/views/estimates/show.html.slim'
```

Comment:
there are a few lines in this file that fail the slim linting process, they are lines that cannot be broken up e.g a URL so ignoring the file is required as there is no fix for the linting issue

Reply:
To be honest, excluding specific files from the linter is a bit of a hack. Ideally we should be saying the rule is worth enforcing (in which case we should move long URLs/SVGs to helper methods so that we don't violate the rule), or that it's not worth enforcing because there are valid reasons to break it (in which case we should disable it entirely or disable it for specific lines with a `/ slim-lint:disable`). I think the footer exclusion is best thought of as a documented piece of tech debt to be resolved when we get a minute.